ansible/deploy_db.yml

```diff
@@ -99,3 +99,32 @@
     - celery
   roles:
     - {role: java, tags: java}
+
+# Disables transparent huge pages (THP)
+- name: Disable THP
+  become: true
+  hosts:
+    - postgresql
+    - couchdb
+    - redis
+    - elasticsearch
+    - rabbitmq
+    - zookeeper
+    - riakcs
+    - formplayer
```

Comment:
if this is in general good for database access, then it may also be good for formplayer due to its frequent access to sqlite. though it's not clear to me what the tradeoffs are for other things like running a normal web server